data/json/skills.json

```diff
@@ -3,176 +3,190 @@
+    "ident": "cooking",
+    "name": "cooking",
+    "description": "Your skill in combining food ingredients to make other, tastier food items.  It may also be used in certain chemical mixtures and other, more esoteric tasks.",
+    "tags": [ "crafting_skill" ]
   },
   {
     "type": "skill",
-    "ident": "archery",
-    "name": "archery",
-    "description": "Your skill in using bow weapons, from hand-carved self bows to complex compound bows.  Quiet and effective, they require strength of body and sight to wield, and are not terribly accurate over a long distance.",
-    "tags": [ "combat_skill" ]
+    "ident": "tailor",
+    "name": "tailoring",
+    "description": "Your skill in the craft and repair of clothing, bags, blankets and other textiles.  Affects knitting, sewing, stitching, weaving, and nearly anything else involving a needle and thread.",
+    "tags": [ "crafting_skill" ]
   },
   {
     "type": "skill",
-    "ident": "bashing",
-    "name": "bashing weapons",
-    "description": "Your skill in fighting with blunt weaponry, from rocks and sticks to baseball bats and the butts of rifles.  Skill increases damage, and higher levels will improve the accuracy of an attack.",
-    "tags": [ "combat_skill" ]
+    "ident": "survival",
+    "name": "survival",
+    "description": "Your skill in surviving the wilderness, and in crafting various basic survival items.  This also covers your ability to skin and butcher animals for meat and hides.",
+    "tags": [ "crafting_skill" ]
   },
   {
     "type": "skill",
-    "ident": "cutting",
-    "name": "cutting weapons",
-    "description": "Your skill in fighting with weaponry designed to cut, hack and slash an opponent.  Lower levels of skill increase accuracy and damage, while higher levels will help to bypass heavy armor and thick hides.",
-    "tags": [ "combat_skill" ]
+    "ident": "electronics",
+    "name": "electronics",
+    "description": "Your skill in dealing with electrical systems, used in the craft and repair of objects with electrical components.  This skill is an important part of installing and managing bionic implants.",
+    "tags": [ "crafting_skill" ]
   },
   {
     "type": "skill",
-    "ident": "dodge",
-    "name": "dodging",
-    "description": "Your ability to dodge an oncoming threat, be it an enemy's attack, a triggered trap, or a falling rock.  This skill is also used in attempts to fall gracefully, and for other acrobatic feats.  The first number shown includes modifiers, and the second does not."
+    "ident": "archery",
+    "name": "archery",
+    "description": "Your skill in using bow weapons, from hand-carved self bows to complex compound bows.  Quiet and effective, they require strength of body and sight to wield, and are not terribly accurate over a long distance.",
+    "tags": [ "combat_skill", "ranged_skill" ]
```

Comment:
If I'm reading your code right, the game looks only for the first hit of tag to determine which category skill in question belongs to, right? If so, the second tag isn't taken into consideration at all?